www/%username/donate.spt

```diff
@@ -125,16 +125,21 @@ full_title = _("Donate to {0} via Liberapay", participant.username)
 
     <h4>{{ _("Can I make a one-time donation?") }}</h4>
     <p>{{ _(
-        "No, but you can stop your donation at any time, and it will de facto "
-        "cease if you don't top up your account when needed."
+        "One-time donations aren't properly supported yet, but you can discontinue "
+        "your donation immediately after the first payment."
     ) }}</p>
 
-    <h4>{{ _("Is this platform legit and secure?") }}</h4>
+    <h4>{{ _("What is this website? I don't recognize it.") }}</h4>
     <p>{{ _(
-        "Yes. Liberapay is run by a non-profit organisation {0}legally registered{1} "
-        "in France, we comply with the European Union's financial regulations, and "
```

Comment:
> we comply with the European Union's financial regulations

This is informative and reassuring for readers. It probably should stay here then.